liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/RunServerMojo.java

```diff
@@ -65,7 +65,14 @@ private void doRunServer() throws MojoExecutionException {
         boolean hasDownstreamProjects = false;
         ProjectDependencyGraph graph = session.getProjectDependencyGraph();
         if (graph != null) {
-            checkMultiModuleConflicts(graph);
+        	
+        	// In a multi-module build, the run server goal will only be run on one project (the farthest downstream) and compile will
+        	// be run on any relative upstream projects. If this current project in the Maven Reactor is not one of those projects, skip it.  
+        	List<MavenProject> relevantProjects = getRelevantMultiModuleProjects(graph);
+        	if (!relevantProjects.contains(project)) {
+        		getLog().info("\nSkipping module " + project.getArtifactId() + " which is not configured for dev mode.\n");
```

Comment:
I think this should say "which is not configured for the run goal." rather than dev mode.